SQL/Archive/19.0/2017-10-30-removeFinalRadiologicalReview.sql

```diff
@@ -4,3 +4,7 @@ DROP TABLE IF EXISTS `final_radiological_review`;
     
 DELETE FROM LorisMenu where Label = "Radiological Reviews";
 update LorisMenu set OrderNumber = (OrderNumber-1) where Parent = 3;
+
+delete from permissions where code = "edit_final_radiological_review";
+delete from permissions where code = "view_final_radiological_review";
+update permissions set permID = (permID-2) where permID > 22;
```

Comment:
I don't think we should delete these from existing projects by default, because users might have moved the module to projects/.. it makes more sense to me to just remove them from the default schema for new installs

Reply:
@driusan Ok, I will back it to normal